binaries/hssi/ethernet/hssicommon.py

```diff
@@ -53,8 +53,10 @@
 # mailbox register poll interval 1 microseconds
 HSSI_POLL_SLEEP_TIME = 1/1000000
 
-# mailbox register poll timeout 500 microseconds
-HSSI_POLL_TIMEOUT = 5/10000
+# Mailbox register poll timeout 1 second. Responses are typically much
+# faster than one second, but in rare cases can take a while. No reason
+# to make this short since timeout is generally a failure.
+HSSI_POLL_TIMEOUT = 1
```

Comment:
In the Linux Kernel the units of the timeout are usually given in the macro name
HSSI_POLL_TIMEOUT_MS or HSSI_POLL_TIMEOUT_SEC would be clearer.